invenio_cli/helpers/commands.py

```diff
@@ -182,11 +192,18 @@ def services(self, force):
                 'pipenv', 'run', 'invenio', 'index', 'queue', 'init', 'purge']
             subprocess.run(command, check=True)
 
-        # If re-run, this doesn't throw an error
+            self.cli_config.update_services_setup(False)
+
+        if self.cli_config.get_services_setup():
```

Comment:
Minor: [Personal biased] I do not like bare returns.

What about:

``` python
if not self.cli_config.get_services_setup():
    [... All code inside this block ...]
```

Reply:
Sure. Would be closer to what was did for `containerize` too.